packages/elements/src/icons/ui/IconsUi.stories.tsx

```diff
@@ -1,83 +1,122 @@
   stenaArrowRight,
   stenaArrowUp,
-  stenaArrowDown,
+  stenaArrowWideDown,
   stenaArrowWideLeft,
   stenaArrowWideRight,
   stenaArrowWideUp,
-  stenaArrowWideDown,
+  stenaBell,
+  stenaBellFilled,
   stenaBusinessAgreement,
-  stenaBusinessCustomer,
   stenaBusinessCard,
   stenaBusinessClaim,
+  stenaBusinessCustomer,
   stenaBusinessInvoice,
   stenaBusinessNSLH,
-  stenaCalendarManage,
-  stenaExpand,
   stenaBusinessSignature,
-  stenaTrash,
-  stenaTimes,
-  stenaTimesCircle,
-  stenaSliders,
-  stenaSearch,
-  stenaRefresh,
-  stenaPlus,
-  stenaMinus,
-  stenaPlusCircle,
-  stenaMinusCircle,
-  stenaPen,
-  stenaHamburger,
+  stenaCalendar,
+  stenaCalendarFilled,
+  stenaCalendarManage,
+  stenaCheck,
+  stenaCheckCircle,
+  stenaCheckCircleFilled,
+  stenaClock,
+  stenaClockFilled,
   stenaCopy,
   stenaCopyDocument,
+  stenaDownload,
+  stenaExclamationCircle,
+  stenaExclamationCircleFilled,
+  stenaExclamationTriangle,
+  stenaExclamationTriangleFilled,
+  stenaExpand,
+  stenaHamburger,
+  stenaInfoCircle,
+  stenaInfoCircleFilled,
+  stenaIntermodal,
   stenaLogout,
   stenaMail,
-  stenaSwitch,
+  stenaMinus,
+  stenaMinusCircle,
   stenaMobile,
+  stenaPen,
+  stenaPhone,
+  stenaPlus,
+  stenaPlusCircle,
+  stenaRefresh,
   stenaSailingGate,
   stenaSailingLocation,
   stenaSailingOnQuay,
   stenaSailingOnRoute,
   stenaSailingShip,
   stenaSailingTrain,
-  stenaCheck,
-  stenaCalendar,
-  stenaBell,
-  stenaBellFilled,
-  stenaExclamationTriangle,
-  stenaExclamationTriangleFilled,
-  stenaExclamationCircle,
-  stenaExclamationCircleFilled,
-  stenaInfoCircle,
-  stenaInfoCircleFilled,
-  stenaCheckCircle,
-  stenaCheckCircleFilled,
-  stenaClock,
-  stenaClockFilled,
-  stenaArrowCircleDown,
-  stenaArrowCircleLeft,
-  stenaArrowCircleRight,
-  stenaArrowCircleUp,
-  stenaCalendarFilled,
+  stenaSearch,
+  stenaSliders,
+  stenaSortDown,
+  stenaSortUp,
+  stenaSwitch,
+  stenaTimes,
+  stenaTimesCircle,
+  stenaTrash,
   stenaUser,
   stenaUserCircle,
-  stenaPhone,
-  stenaSortUp,
-  stenaSortDown,
-  stenaDownload,
-  stenaIntermodal,
 } from "./IconsUi";
+import { Box, Txt } from "@stenajs-webui/core";
+import { Icon } from "../../components/ui/icon/Icon";
 
 export default {
   title: "elements/Icons/UI",
 };
 
+export const AllIconsDemo = () => (
+  <Box
+    indent
+    spacing
+    style={{
+      display: "grid",
+      gap: "8px",
+      gridTemplateColumns: "repeat(auto-fill, minmax(80px, 1fr))",
+      background: "#f0f1f3",
+    }}
+  >
+    {Object.keys(allIcons).map((iconName) => (
+      <Box
+        gap
+        indent
+        borderRadius={8}
+        style={{ aspectRatio: "1", cursor: "pointer" }}
+        background={"rgb(255, 255, 255)"}
+        justifyContent={"center"}
+        alignItems={"center"}
+        hoverBackground={"rgb(255, 212, 59)"}
+        onClick={() => navigator.clipboard.writeText(iconName)}
+      >
+        <Icon key={iconName} icon={allIcons[iconName]} size={24} />
```

Comment:
I understand that ui-500 is the default icon colour, but I don't like it. I would like at least ui-600 for better contrast.

Reply:
I agree!